Makefile

```diff
@@ -2130,16 +2133,9 @@ EXTRA_CFLAGS += -DMARK_KERNEL_PFU
 else ifeq ($(ARCH), x86_64)
 EXTRA_CFLAGS += -mhard-float
 EXTRA_CFLAGS += -DMARK_KERNEL_PFU
-else ifeq ($(ARCH), arm)
```

Comment:
I'd rather do something like:

```
else ifeq ($(ARCH),arm)
EXTRA_CFLAGS += -mfloat-abi=softfp
else ifeq ($(ARCH), arm64)
EXTRA_CFLAGS += -mfloat-abi=hard
else
$(error ARCH: unknown type $(ARCH))
endif
```

Reply:
Except that's also wrong. Building with softfp on my Raspberry Pi 2B on Ubuntu causes the driver to BUG: FP instruction issued in kernel mode with FP unit disabled. See #804 .

Reply:
Okay, so what's expected there?

Reply:
In my situation, it works by passing no -mfloat-abi argument at all. Prior to 3325543 the -mfloat-abi flag was only added for CONFIG_MP_VHT_HW_TX_MODE.

Reply:
Okay, that's a little confusing then...  If `CONFIG_MP_VHT_HW_TX_MODE` isn't set, then no FP instructions should be emitted into the code stream... and therefore the setting of `-mfloat-abi` should be irrelevant.